.devcontainer/dev.Dockerfile

```diff
@@ -49,9 +48,9 @@ RUN groupadd --gid "$USER_GID" "$USERNAME" \
 COPY .devcontainer/krb5-redhat.conf /etc/krb5.conf.d/
 
 # Preinstall dependencies
-COPY ./requirements.txt ./requirements-dev.txt /tmp/doozer/
+COPY ./ /tmp/doozer/
 RUN pushd /tmp/doozer \
-  && sudo -u "$USERNAME" pip3 install --user -r ./requirements.txt -r requirements-dev.txt \
-  && popd && rm -rf /tmp/doozer
+ && sudo -u "$USERNAME" pip3 install --user -r ./requirements.txt -r requirements-dev.txt ./ \
```

Comment:
Needs `chown`.

Reply:
done.
it built without it on my docker, but :shrug: 
thanks for checking, i wanted to make sure not to break you. can you confirm it works with the `chown` added?

Reply:
Yes, it works now. I guess probably it is because my local docker env doesn't have user namespace enabled.